MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1159,6 +1159,10 @@
         uint32_t maxStorage = 0, maxUniform = 0;
         bool singleTexelStorage = true, singleTexelUniform = true;
         mvkEnumerateSupportedFormats({0, 0, VK_FORMAT_FEATURE_UNIFORM_TEXEL_BUFFER_BIT | VK_FORMAT_FEATURE_STORAGE_TEXEL_BUFFER_BIT}, true, [&](VkFormat vk) {
+            // MTLDevice minimumLinearTextureAlignmentForPixelFormat with MTLPixelFormatInvalid will cause as a failed assertion on iOS 13.x and iPad OS 13.x
+            if ( vk == VK_FORMAT_UNDEFINED )
+              return false;
+          
```

Comment:
MoltenVK doesn't use hanging if and for statements. I'll pull in anyway and fix it here.

Reply:
Sorry for that, I could have added this check at the call site, feel free to modify so it follows you standards. We just had a few customers with iOS Beta 13 and they reported a crash on application start. This line fixed it. It seems like Metal has extra assertions on iOS 13 when used with wrong parameters.

Reply:
No worries. Yeah...Apple often adds new validation operations on each release of Metal. Thanks for catching and fixing this.